src/mapReducers.js

```diff
@@ -26,6 +26,40 @@ function resolveKey(action, keySelector) {
   return resolvedKey || TARGET_ALL_REDUCERS;
 }
 
+function calculateNewTargetAllState(state, action, resolveReducer) {
```

Comment:
Also, document that we are doing this in order to preserve the old references when the state doesn't change.